runtime/doc/api.txt

```diff
@@ -1251,19 +1251,21 @@ nvim_replace_termcodes({str}, {from_part}, {do_lt}, {special})
 
                                                 *nvim_select_popupmenu_item()*
 nvim_select_popupmenu_item({item}, {insert}, {finish}, {opts})
-    Selects an item in the completion popupmenu.
+    Selects an item in the completion popup menu.
 
-    If |ins-completion| is not active this API call is silently ignored.
-    Useful for an external UI using |ui-popupmenu| to control the popupmenu
-    with the mouse. Can also be used in a mapping; use <cmd> |:map-cmd| to
-    ensure the mapping doesn't end completion mode.
+    If neither |ins-completion| nor |cmdline-completion| popup menu is active
+    this API call is silently ignored. Useful for an external UI using
+    |ui-popupmenu| to control the popup menu with the mouse. Can also be used
+    in a mapping; use <Cmd> |:map-cmd| to ensure the mapping doesn't end
+    completion mode.
```

Comment:
as you tested it works with lua function mappings, we could mention here that those work as well.